src/gsy_e/gsy_e_core/util.py

```diff
@@ -443,21 +475,39 @@ def get_market_maker_rate_from_config(next_market, default_value=None, time_slot
                 raise e
         return find_object_of_same_weekday_and_time(GlobalConfig.market_maker_rate,
                                                     time_slot)
-    else:
-        return GlobalConfig.market_maker_rate
+    return GlobalConfig.market_maker_rate
+
+
+def get_feed_in_tariff_rate_from_config(next_market, default_value=None, time_slot=None):
+    """Get feed in tariff rate from config."""
+    if next_market is None:
+        return default_value
+    if isinstance(GlobalConfig.FEED_IN_TARIFF, dict):
+        if time_slot is None:
+            try:
+                time_slot = next_market.time_slot
```

Comment:
`FutureMarkets` objects also have a `time_slot` attribute, so this check will not be triggered.

Reply:
I haven't considered future market for now for SavingsKPI and would leave it for time being to not make it too complex, and once SavingsKPI has been fully tested for spot market, future market could be integrated late as per need

Reply:
I understood that, but the above check is not checking it the `next_market` is a future market. 
The area class has a `is_market_future` that could be handy, but only if you have access to the area.

Reply:
You can also check whether the `next_market.time_slot is None`, up until now this is only possible in the future market